storage/ping/chunk_ping_bing.com.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
kindly restore all **ping chunks** from the original code. We don't wanna delete these in each PR, they need to be present at the time of writing tests!

Reply:
They are not in the original code